packages/noya-state/src/selectors.ts

```diff
@@ -103,6 +115,16 @@ export const getSelectedLayersExcludingDescendants = (
   );
 };
 
+export const getSelectedStyles = (state: ApplicationState): Sketch.Style[] => {
+  const currentTab = getCurrentTab(state);
+
+  return currentTab === 'canvas'
+    ? getSelectedLayers(state).flatMap((layer) =>
+        layer.style ? [layer.style] : [],
+      )
+    : getSelectedLayerStyle(state).flatMap((style) => style.value);
```

Comment:
Can this be a regular `.map`? We only need `flatMap` to flatten arrays - in the first case I use it to remove undefined values, but I don't think we have that in this second case